dcos/rpcclient.py

```diff
@@ -145,7 +144,8 @@ def http_req(self, method_fn, path, *args, **kwargs):
                 request_method=e.response.request.method,
                 request_url=e.response.request.url,
                 json_body=json_body)
-            raise DCOSException(message)
+
+            raise DCOSException(message) from e
```

Comment:
The stacktrace is required for Shakedown. Right now debugging is super heard. Actually, `RpcClient.http_req` swallows more information since the actual method is passed.